StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/ReadabilityResources.resx

```diff
@@ -534,4 +534,13 @@
   <data name="SX1101Title" xml:space="preserve">
     <value>Do not prefix local calls with 'this.'</value>
   </data>
+  <data name="SA1140Description" xml:space="preserve">
+    <value>Code should be wrapped to a maximum line length. See stylecop.json</value>
+  </data>
+  <data name="SA1140MessageFormat" xml:space="preserve">
+    <value>Maximum line length exceeded</value>
```

Comment:
I think it would be good to change this to "Maximum line length of {0} exceeded" and make it include the set line length.